### PR TITLE
[#40429] realtime notification

### PR DESCRIPTION
**Pull request by trunghq-2593**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/???)

## WHAT (optional)
- real time for partner when customer book room

## HOW
- Notification Event laravel 
- Pusher
-- show real time notify when customer order
-- partner can mark each notity or mark all as read

## Evidence (Screenshot or Video)
Screenshot from 2021-08-25 21-25-06

### CODE REVIEW

**Comment by trunghq-2593** [comment]
em/mình làm chức năng này hơi quá file tí review giúp em/mình với ạ. Thanks u

**Comment by trunghq-2593** [comment]
ready

File: app/Notifications/OrderNotification.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+                    ->line('The introduction to the notification.')
+                    ->action('Notification Action', url('/'))
+                    ->line('Thank you for using our application!');
```
Comment (ghost) [inline app/Notifications/OrderNotification.php]: indent 4 space

Comment (zunokie) [reply app/Notifications/OrderNotification.php]: oke b

**Comment by trunghq-2593** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**